orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/constant/OrientDBConstant.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.datastore.ogm.orientdb.constant;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Sergey Chernolyas <sergey.chernolyas@gmail.com>
+ */
+
+public class OrientDBConstant {
+
+	public static final String DATE_FORMAT = "yyyy-MM-dd'Z'";
+	// public static final String DATETIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+	public static final String DATETIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
```

Comment:
Is it something that you really need to define in OrientDB? Keep in mind the different user might want to use different date format